blockmanager.go

```diff
@@ -761,6 +761,14 @@ func (b *blockManager) getUncheckpointedCFHeaders(
 func (b *blockManager) getCheckpointedCFHeaders(checkpoints []*chainhash.Hash,
 	store *headerfs.FilterHeaderStore, fType wire.FilterType) {
 
+	// We fetch the genesis header to use for verifying the first received
+	// interval.
+	genesisHeader, err := store.FetchHeaderByHeight(0)
```

Comment:
cache this in `blockManager` during `Start()`?

Reply:
Good idea! Added to `newBlockManager`.